content/article/20170912-prediction-of-huawei-sales-of-vidhance-models.md

```diff
@@ -0,0 +1,135 @@
+
+Both the Mate 9 and the P10 comes in several variants. The variants are
+|Name| Running Vidhance stabilization?|
+|---|---|
+|Mate 9| Yes|
+|Mate 9 Pro| Yes |
+|Mate 9 Porsche design| Yes |
+| Mate 9 Lite | No |
+| P10 | Yes |
+| P10 Plus | Yes |
+| P10 Lite | No |
+
+For the remainder of the article, unless other is stated, by referring to "Mate 9" or "P10" I refer to all the variants of that phone, except for the "Lite" variant that does not run Vidhance software. 
+
+#### Summary
+Vidhance is currently beeing used in the following product-lines by Huawei (and Honor)
+* Mate 9
+* P10
+* Honor 9
+
+Vidhance is beeing run in all "flagship"-phones released by Huawei for the last 12 months, and it's also used in the latest "flagship"-phone by "Honor"-subsidiary. Honor have earlier in the year released another flagship phone ("Honor V9, AKA Honor 8 Pro). The appearance of Vidhance in the Honor-brand is major, as it confirms Honor beeing a part of the license-agreement with Huawei. Honor is by itself a large global smartphone brand, with big volumes for their popular models.
+
+## Detailed sales for specific models
+This is where things become hard. Huawei seldom provide detailed statistics of sales volume for it's models. Bits and pieces are sometimes dropped through annual reports or an occasional press release. So let's start with some known facts.
+
+### P8 Lite
+We know that by Februar 2016, the P8 Lite had shipped above 10 million units, since it's release in April 2015. The entire P8 series (all P8 models) had by the same time shipped above 16 million units [(source)](http://consumer.huawei.com/en/press/news/2016/hw-472293/). Huawei have provided a detailed monthly shipment chart for this specific model, which is quite unique, and therefore an important data source.
+
+![P8 Lite shipment trend](http://img.c.huaweistatic.com/content/dam/huawei-cbg-site/en/mkt/press/news/2016/31609-1.jpg "P8 Lite shipment trend")
+
+### P9
+Although the P8 Lite managed to sell above 10 million units, P9 was credited by Huawei as beeing the first _"flagship"_ device to do so.
+
+### General sales
+Here is a table trying to give every known sales volumes by offical (or at least credible) sources, for all Huawei and Honor models that might be relevant for predicting the sales volumes of the models running Vidhance. _Italic data points_ are estimations based upon other __known datapoints, marked in bold__. Paranthesis mark the source of information for the datapoint.
+
+| Model | Released | Month 1 | Month 2 | Month 3  | Month 4  | Month 5  | Month 6  | Month 7  | Month 8  | Month 9  | Month 10  | Month 11  | Month 12|
```

Comment:
This table is too wide for the page. I might find a way to fix it but we should consider rearranging the data somehow.